util/tune/current_settings.rb

```diff
@@ -216,15 +220,12 @@ def puppetdb_java_args
 # TODO: update to eliminate the use of exclusions
 #   See https://tickets.puppetlabs.com/browse/SLV-531
 #
-def get_conf_parameter(file, parameter, exclusions = DEFAULT_EXCLUSION)
+def get_conf_parameter(file, parameter)
   value = if File.exist? file
-
-            # this pattern extracts the value from the result with the '\K'
-            match_pattern = /#{parameter} = \K[^\s]+/
-
-            # eliminate comments, search for the parameter, apply exclusions, extract the value
-            File.open(file).grep_v(/^#/).grep(/#{parameter}/).grep_v(exclusions)[0].match(match_pattern)
-
+            conf = File.read(file)
```

Comment:
This will throw an exception if the file doesn't exist.  Is that the expected behavior, or should the error be eaten to allow the `NA` value to be returned by this method?

Reply:
It's inside an if file.exists? block, so the file must exist.